wandb/sdk/wandb_run.py

```diff
@@ -3117,12 +3119,13 @@ def log_artifact(
             type: (str) The type of artifact to log, examples include `dataset`, `model`
             aliases: (list, optional) Aliases to apply to this artifact,
                 defaults to `["latest"]`
+            tags: (list, optional) Tags to apply to this artifact, if any.
 
         Returns:
             An `Artifact` object.
         """
         return self._log_artifact(
-            artifact_or_path, name=name, type=type, aliases=aliases
+            artifact_or_path, name=name, type=type, aliases=aliases, tags=tags
```

Comment:
nit: this is enough parameters to justify adding a trailing comma and reformatting to have each parameter on its own line

Reply:
agreed